code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -270,6 +270,10 @@
 
 	msg += "</span>"
 
+	if(HAS_TRAIT(user, TRAIT_SPIRITUAL) && !appears_dead && (src != user) && src.mind.isholy)

```

Comment:
If he is dead maybe a bad moodlet :^)

Reply:
`appears_dead` needs to be in an overarching check rather than be checked 3 different times in the same proc.